sdcm/coredump.py

```diff
@@ -308,6 +316,16 @@ def systemd_version(self):
             self.log.warning("failed to get systemd version:", exc_info=True)
         return systemd_version
 
+    @contextmanager
+    def hard_link_corefile(self, corefile):
+        hard_links_path = Path(corefile).parent / 'hardlinks'
+        link_path = hard_links_path / Path(corefile).name
+        self.node.remoter.sudo(f'mkdir -p {hard_links_path}', ignore_status=True)
+        self.log.debug(f'doing: ln {corefile} {link_path}')
+        self.node.remoter.sudo(f'ln {corefile} {link_path}', ignore_status=True)
+        yield link_path
+        self.node.remoter.sudo(f'rm -f {link_path}', ignore_status=True)
```

Comment:
I'm not sure if we should ignore statuses her. What is the reason?

Reply:
Best effort for a secondary goal.

Reply:
worse case we'll be filling the disk, and we get to that, we are in a shity situattion already :)